.github/workflows/Build.yml

```
@@ -730,3 +730,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./ci
+
+  #65
+  xyce-linux:
+    runs-on: "ubuntu-20.04"
+    env:
+      PACKAGE: "sim/xyce"
```

```suggestion
    env:
      CONDA_BUILD_PREPARE_REV: "aj-fix-describe"
      PACKAGE: "sim/xyce"
```

Please set this variable to use `conda-build-prepare` WIP branch that is able to handle "misnamed" tags for now to build Xyce until https://github.com/hdl/conda-build-prepare/issues/1 gets fixed. I've just pushed a commit to conda-eda's master` to enable specifying it so please first rebase this PR on top of it.